apps/codeforafrica/src/components/AccoladeBadgeList/AccoladeBadgeList.js

```diff
@@ -0,0 +1,50 @@
+import { Grid } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import AccoladeBadge from "@/codeforafrica/components/AccoladeBadge";
+
+const AccoladeBadgeList = React.forwardRef(function AccoladeBadgeList(
+  props,
+  ref
+) {
+  const { badges, ...other } = props;
+
+  if (!badges?.length) {
+    return null;
+  }
+  return (
+    <Grid
+      container
+      spacing={{ xs: 0, sm: "5px", md: "29px" }}
+      sx={{
+        display: "flex",
+        flexDirection: { xs: "column", sm: "row" },
+        alignItems: "center",
+        width: "100%",
+      }}
+      ref={ref}
+      {...other}
+    >
+      {badges.map((item) => (
+        <Grid item xs={12} sm={6} md={3}>
+          <AccoladeBadge name={item.name} date={item.date} />
```

Comment:
```suggestion
          <AccoladeBadge {...item} />
```

Unless there are props you don't want to pass.